style.css

```diff
@@ -0,0 +1,588 @@
+body {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+  }
+  
+  header {
+    position: fixed;
+    display: grid;
+    width: 100vw;
+    grid: 56px / 60px 150px auto 210px;
+    background-color: white;
+    z-index: 1;
+  }
+  
+  .menu-button-container {
+    display: flex;
+  }
```

Comment:
It would be better to put a one-line space between two CSS code blocks.

Reply:
I recommend adding a one-line space between two CSS code blocks for better readability and organization.